Makefile

```diff
@@ -2,50 +2,24 @@
 TARGET_OS ?= linux
 
 # When developing locally, change this to whatever fqdn you are using for 127.0.0.1
-VIRTUAL_HOST ?= localhost
+DOMAIN ?= localhost
```

Comment:
How about `TLS_DOMAIN`?

Reply:
I can't wait for this to be merged... any ETA in mind ? :)

Reply:
@Shaltz In case you're referring to me, I'm just a random dude with no connection to the maintainers other than checking out their project - @antoinegauvain and @eranchetz are prob in a better position to answer ^^